vocabularies/Org.OData.Aggregation.V1.xml

```diff
@@ -311,19 +343,25 @@
       </Term>
 
       <Term Name="LeveledHierarchy" Type="Collection(Edm.PropertyPath)" Nullable="false" AppliesTo="EntityType ComplexType">
-        <Annotation Term="Core.Description" String="Defines a leveled hierarchy [OData-Aggr, section 5.5.1]" />
+        <Annotation Term="Core.Description" String="Defines a leveled hierarchy [OData-Data-Agg-v4.0, section 5.5.1]" />
       </Term>
 
       <Term Name="RecursiveHierarchy" Type="Aggregation.RecursiveHierarchyType" Nullable="false" AppliesTo="EntityType">
-        <Annotation Term="Core.Description" String="Defines a recursive hierarchy [OData-Aggr, section 5.5.2]" />
+        <Annotation Term="Core.Description" String="Defines a recursive hierarchy [OData-Data-Agg-v4.0, section 5.5.2]" />

```

Comment:
Should this `[...]` reference be a hyperlink?

Reply:
No.